mmv1/products/compute/api.yaml

```diff
@@ -12944,6 +12944,13 @@ objects:
                   - snapshot_schedule_policy.0.snapshot_properties.0.guest_flush
                 description: |
                   Whether to perform a 'guest aware' snapshot.
+              - !ruby/object:Api::Type::String
+                name: 'chainName'
+                allow_empty_object: true
```

Comment:
`allow_empty_object` field can be removed.. This only [operates on nested fields](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/type.rb#L69-L77). What was your intention in adding this?